snapshot-labs/sx-monorepo
# fix: add support for showing approval voting results

### Summary

Add support for showing vote result for approval voting type (in the proposal's "vote" tab)

### How to test

1. Go to a proposal with `approval` voting type (e.g. http://localhost:8080/#/s:pistachiodao.eth/proposal/0x08c3bd2960700525770a1d634f8599ba967e55fcc05b6c1649d984d88253769d/votes)
2. It should show all the user's choices
3. It should show the full choices string in the tooltip, in case the string is truncated inside the table

![Screenshot 2024-03-15 at 14 39 57](https://github.com/snapshot-labs/sx-monorepo/assets/495709/6a2daaa8-3797-4696-abdc-6f22337be731)

It should keep showing the old format for the existing vote type:

Single choice:
![Screenshot 2024-03-15 at 14 40 03](https://github.com/snapshot-labs/sx-monorepo/assets/495709/c6a98b10-26f1-4023-b4af-e070113fc1b2)

Basic:
![Screenshot 2024-03-15 at 14 40 09](https://github.com/snapshot-labs/sx-monorepo/assets/495709/5b9df878-cb3a-42bf-bd7c-519852835b81)

The VotesModal should also show the same choice text, depending on voting type, albeit may need some UI tweaking

![Screenshot 2024-03-15 at 15 41 17](https://github.com/snapshot-labs/sx-monorepo/assets/495709/6dbe7bff-04d5-4d8f-bbbf-282d5e05bab8)
